_scss/defold.scss

```diff
@@ -18,6 +18,113 @@
 	--defold-green: #57ab5a;
 }
 
+/*******************************************************************************
```

Comment:
I wonder if maybe we should start splitting up defold.scss? It's getting quite large. Sure, in the end we combine all .scss files into a mega css but from a workflow perspective maybe it would be nice to have a youtube.scss?